### PR TITLE
Enable MyBinder in `HybridQ`

### PR DESCRIPTION
**Pull request by s-mandra**

### CODE REVIEW

**Comment by s-mandra** [comment]
`.github/workflows/` must be changed to take into account the fact that `Dockerfile` now pulls directly from `smandra/hybridq-baseline` (`Dockerfile` is using `smandra/hybridq-baseline` instead of `smandra/hybridq-baseline:[python version]`).

**Comment by s-mandra** [comment]
> `.github/workflows/` must be changed to take into account the fact that `Dockerfile` now pulls directly from `smandra/hybridq-baseline` (`Dockerfile` is using `smandra/hybridq-baseline` instead of `smandra/hybridq-baseline:[python version]`).

Fixed.

**Comment by s-mandra** [comment]
MyBinder can be tested at [https://mybinder.org/v2/gh/nasa/hybridq/mybinder](url).

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # HybridQ: A Hybrid Simulator for Quantum Circuits
 
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/nasa/hybridq/main)
```
Comment (teubert) [inline README.md:3]: This link doesn't work for me. I get this error:
`
pull access denied for hybridq-baseline, repository does not exist or may require 'docker login': denied: requested access to the resource is deniedBuilt image, launching...
Failed to connect to event stream
`

Comment (s-mandra) [reply README.md:3]: The link doesn't work yet as it points to the `main` branch. Use https://mybinder.org/v2/gh/nasa/hybridq/mybinder for testing.

**Review by teubert — APPROVED** [review]
works great!

**Pull request merged**